testing/constraints-3.7.txt

```diff
@@ -11,3 +11,4 @@ setuptools==40.3.0
 rsa==3.1.4
 aiohttp==3.6.2
 requests==2.20.0
+pyjwt==2.0
```

Comment:
Is there a way to perform testing with this pyjwt not added to the test? Want to make sure existing users are not broken

Reply:
ran a manual test to verify existing customers won't be broken. In future we will update our testing so that automated testing can handle this.